normandy/selfrepair/static/js/self_repair.js

```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill'
```

Comment:
Moving this here just moves the problem elsewhere, and while we aren't likely to need to import this from test code, we might. I think there are better approaches, like getting babel to inject this automatically, or including it as a webpack entry point (I'm not sure if/how either of those work at the moment though)